content/en/blog/grpconf-2024-videos.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Highlights in gRPConf 2024:Customer Showcase, Developer Engagement, Birds of Feathers Discussions and more.
+date: 2024-07-03
+---
+
+We loved hosting gRPConf 2024 last month in Sunnyvale, California. Thank you to 
```

Comment:
it's not last month. "This year" or "This summer".

Reply:
done